migrations/sql/V2021.07.09_1359__WorkflowRunReports.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE orderlyweb_workflow_run_reports
```

Comment:
Please add this table to [README.md](https://github.com/vimc/orderly-web/blob/master/README.md#orderlyweb-database-schema)